File: java/src/main/java/com/iteso/nintendo/attacks/ImpactThunder.java
package com.iteso.nintendo.attacks;

import com.iteso.nintendo.attacks.Attack;

public class ImpactThunder implements Attack {
    static int damage;
    private static final String name = "Impact Thunder";

    @Override
    public String attackName() {
        return name;
    }

    @Override
    public int attackDamage() {
        return this.damage;
    }

    @Override
    public void setDamage(int damage) {
        this.damage = damage;
    }
}

File: python/ImpactThunder.py
class ImpactThunder:
    def __init__(self):
        self.damage = None
        self.name = 'Impact Thunder'

    def attackName(self):
        return self.name

    def attackDamage(self):
        return self.damage

    def setDamage(self, damage):
        self.damage = damage
